app/views/curation_concerns/file_sets/upload/_form_fields.html.erb

```diff
@@ -12,14 +12,14 @@
                 <span class="btn btn-primary fileinput-button">
                     <i class="glyphicon glyphicon-plus" aria-hidden="true"></i>
                     <span aria-hidden="true">Select files...</span>
-                    <input type="file" name="files[]" multiple />
```

Comment:
we need to pass the upload_set_id to it as we do here: https://github.com/projecthydra-labs/curation_concerns/blob/64eab4d87fac8b88367ebb5e34b4d34ad7ce6716/app/views/curation_concerns/file_sets/upload/_form.html.erb